docs/AQL/master03-syntax.adoc

```diff
@@ -68,6 +68,17 @@ AQL syntax supports parameterisation. AQL parameters are used to indicate criter
 
 The parameters can be EHR specific (such as the parameter of EHR identifier or EHR creation date time), archetype specific (such as an archetype identifier, or archetype constraints), or template specific (such as a template identifier or template constraints).
 
+=== Parameter syntax
+
+A parameter always starts with a dollar sign `$` followed by the parameter name, e.g. `$ehrUid`. The parameter name can consist of letters, digits and underscores. It cannot have spaces and it cannot be an AQL reserved word.
+
+A parameter can be used for any criteria values within an AQL statement, e.g.:
+
+* within a predicate: `[$archetypeId]`, `[at0003, $nameValue]`, `[ehr_id/value=$ehrId]`
+* withing a WHERE criteria: `o/data[at0001]/events[at0006]/data[at0003]/items[at0004]/value/value > $systolicCriteria`
+
+_Note: in the WHERE criteria, when the value of the parameter is not a number or boolean value, it should be substituted with the correspondent quotes added to the value, for instance `o/../.. = $textVariable`, when substituted should look like this `o/../.. = "value"` (single or double quotes could be used)._
```

Comment:
Is this '_Note:' the same as 'NOTE:'? I would probably use the latter, since that is what is used in all the other specs.

Reply:
Is like that, _xxx_ makes it italic, but I like the default format for NOTE:, so will change it.